parse-config.go

```diff
@@ -270,6 +270,26 @@ func (c *SubsysConfig) Lookup(key string) (val string, present bool) {
 	return c.KV[idx].Value, true
 }
 
+// Lookupenv resolves the value of a config parameter.
```

Comment:
More specifics pls... It is not really obvious from the code what it is doing (to me at least)